cogeo_mosaic/backends/dynamodb.py

```diff
@@ -52,9 +52,35 @@ def write(self):
         items = self._create_items()
         self._write_items(items)
 
-    def update(self):
+    def _update_quadkey(self, quadkey: str, dataset: List[str]):
+        """Update quadkey list."""
+        self.table.put_item(Item={"quadkey": quadkey, "assets": dataset})
+
+    def _update_metadata(self, bounds: List[float], version: str):
```

Comment:
Should this also update `self.mosaic_def`? to stay in sync with what is in dynamodb